processor/groupbytraceprocessor/storage_memory.go

```diff
@@ -35,34 +35,33 @@ var _ storage = (*memoryStorage)(nil)
 
 func newMemoryStorage() *memoryStorage {
 	return &memoryStorage{
-		content:                   make(map[string][]pdata.ResourceSpans),
+		content:                   make(map[[16]byte][]pdata.ResourceSpans),
 		metricsCollectionInterval: time.Second,
 	}
 }
 
 func (st *memoryStorage) createOrAppend(traceID pdata.TraceID, rs pdata.ResourceSpans) error {
-	sTraceID := traceID.HexString()
+	bTraceID := traceID.Bytes()
```

Comment:
Since a few releases, we can use traceID directly. But this isn't required for this PR, is it?

Reply:
This will be automatically resolved when we agree on the comment above